core/primitives-core/src/config.rs

```diff
@@ -215,6 +215,19 @@ pub struct ExtCostsConfig {
     /// Cost of getting sha256 per byte
     pub keccak512_byte: Gas,
 
+    /// Cost of getting ripemd160 base
+    pub ripemd160_base: Gas,
```

Comment:
These should be guarded by feature as well. Otherwise backward compat tests fail - https://buildkite.com/nearprotocol/nearcore/builds/5739#ee3b3336-f269-4c34-ac2b-fb736925da7e